tests/functional/models/test_slate_lineup_model.py

```diff
@@ -90,3 +110,50 @@ async def test_get_slate_lineup_no_slates(self, slate_config, slate_lineup_confi
 
         slate_lineup = await SlateLineupModel.get_slate_lineup(slate_lineup_config_id, slate_count=0)
         assert len(slate_lineup.slates) == 0
+
+    @patch.object(SlateConfigModel, 'SLATE_CONFIGS_BY_ID', slate_configs_by_id)
+    @patch('app.models.slate_lineup_config.SlateLineupConfigModel.find_by_id', return_value=slate_lineup_config_model_2)
+    async def test_get_slate_lineup_dedupe_recs(self, slate_lineup_config):
```

Comment:
We're doing a weird combination of patching and seeding in these tests. We've been doing it since before this PR so I don't think we should hold this PR on it, but might be worth a team chat about why we've alighted on this pattern and whether we're doing it because we _want_ to or we're doing it because something is _forcing_ us to that we might be able to fix

Reply:
Agreed. I think it would be nicer if we called `SlateConfigModel.load_slate_configs` before the test, and load it with a json fixture from tests/assets/json.

Reply:
Retro topic? I know we have a set of standards in TS but we have to talk about python or even set up a python linter.